### PR TITLE
Using new translation method for R xgettext

### PR DESCRIPTION
**Pull request by shun2wang**
Close https://github.com/jasp-stats/INTERNAL-jasp/issues/2155

Ignore unnecessary translation strings with `"stop" "warning" "message" "packageStartupMessage"` calls in .R file.

### CODE REVIEW

File: translations/translate.R [hunk 1]
```diff
@@ -13,4 +16,312 @@ if(!exists("rootfolder"))
 
 Sys.setlocale(category = "LC_ALL", locale = "en_US.UTF-8")
 
-try(tools::update_pkg_po(rootfolder))
+
+# refactoring xgettext -> jaspXgettext
```
Comment (vandenman) [inline translations/translate.R:20]: Could you add comments of the form
```r
# start of changes
# end of changes
```
to each changed block of code in both functions?
See for example here: https://github.com/jasp-stats/jaspRegression/blob/3b9646176dfac865c5e96118edd53e53720af97c/R/regressionlinearbayesian.R#L814-L823

That way it'll be easier to update the code in the future.

Comment (shun2wang) [reply translations/translate.R:20]: Sure, I will add this, thanks.

**Comment by shun2wang** [comment]
Comments with first letter capitalize is from JASP(i.e. from me) changes. 
Mostly changes just access internal functions with `tools:::`.

**Comment by shun2wang** [comment]
Hi @boutinb , do you have time look at this (*^_^*)

**Review by boutinb — APPROVED** [review]

**Pull request merged**